vsts/pipelines/_buildTemplate.yml

```diff
@@ -167,6 +167,12 @@ steps:
     scriptPath: ./vsts/scripts/dockerCleanup.sh
   condition: or(eq(variables['TestBuildImages'], 'true'), eq(variables['TestRuntimeImages'], 'true'), eq(variables['TestIntegration'], 'true'))
 
+- task: ShellScript@2
+  displayName: 'Generate release notes'
+  inputs:
+    scriptPath: ./vsts/scripts/generate-release-notes.sh
+  condition: and(succeeded(), eq(variables['PushBuildImages'], 'true'), eq(variables['BuildBuildImages'], 'true'))
```

Comment:
I guess this condition was probably used for your testing? Ideally it should probably be only for our `Oryx-CI` builds?

Reply:
It's fine to have it in other builds, so we can see how it would look like before the CI build happens. There's just going to be a text file with release notes in the build output folder

Reply:
Sure, sounds good